### ISSUE TITLE
Inline Comments - Error 1098, Cannot find ... [ENDT] that closes ... [TEXT]

### ISSUE THREAD

**Issue by Kirides** [fdbozzo/foxbin2prg#26]
It seems there is an Issue with Inline comments, see Code and Log

Code snippet:
```
create cursor (lcTabelle) (;
    SOME_ID N(4),; 
    TEXT C(50),; && This is Any Comment
    SOME_OTHER N(10,2);
    )
```

Log: 
```
2018-07-09T10:28:58  ----------------------------------------------------------------------------------------------------
Error 1098, Cannot find end marker [ENDT] that closes start marker [TEXT] on line 18
identifyexclusionblocks, 7133

Error location ..............................
Method: leistungscontainer.erstelle_cursor, Line 17
> 	TEXT C(50),; && This is Any Comment
Recno: 223
.............................................
FoxBin2Prg: [C:\PORTABLEAPPS\FOXBIN2PRG\FOXBIN2PRG.EXE] (EXE Version: v1.19.51)
```

**Comment by fdbozzo** [comment]
Hi Kirides,

This bug is now fixed on tag [v1.19.51.3](https://github.com/fdbozzo/foxbin2prg/tree/v1.19.51.3)

Please, confirm if it's working for you too.

Thanks for reporting! :D

**Comment by Kirides** [comment]
The error is gone now - thanks for the quick fix. 👍
